
f = {
    -- This caused an interpreter crash at one point
    1[1] = 1
}.block

int = make_stack()
int.stack_push(f [])
int.run
assert(int.errored)
